pkg/daemon/daemon.go

```diff
@@ -2807,3 +2810,80 @@ func cipherOrder() []uint16 {
 
 	return append(first, second...)
 }
+
+type rpmOstreeStatus struct {
+	Deployments []struct {
+		RequestedPackages []string `json:"requested-local-packages"`
+	} `json:"deployments"`
+}
+
+func (dn *Daemon) getUnsupportedPackages() {
+	cmd := exec.Command("rpm-ostree", "status", "--json")
+	output, err := cmd.Output()
+	if err != nil {
+		klog.Errorf("Failed to run rpm-ostree status: %v", err)
+		return
+	}
+
+	var status rpmOstreeStatus
+	if err := json.Unmarshal(output, &status); err != nil {
+		klog.Errorf("Failed to parse rpm-ostree status output: %v", err)
+		return
+	}
+
+	// List of supported packages
+	supportedPackages := map[string]bool{
+		"usbguard":             true,
```

Comment:
Might be good to have this and our supported extensions share a constant somewhere, so we don't have to update both in case we add more packages

Reply:
great- just saw a getSupportedExtensions() in the daemon so will use that.